dune/xt/functions/interfaces/global-flux-function.hh

```diff
@@ -0,0 +1,257 @@
+// This file is part of the dune-xt-functions project:
+//   https://github.com/dune-community/dune-xt-functions
+// Copyright 2009-2017 dune-xt-functions developers and contributors. All rights reserved.
+// License: Dual licensed as BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+//      or  GPL-2.0+ (http://opensource.org/licenses/gpl-license)
+//          with "runtime exception" (http://www.dune-project.org/license.html)
+// Authors:
+//   Felix Schindler (2017)
+
+#ifndef DUNE_XT_FUNCTIONS_INTERFACES_GLOBAL_FLUX_FUNCTION_HH
+#define DUNE_XT_FUNCTIONS_INTERFACES_GLOBAL_FLUX_FUNCTION_HH
+
+#include "localizable-function.hh"
+
+namespace Dune {
+namespace XT {
+namespace Functions {
+
+
+template <class OtherEntityImp, class GlobalFluxFunctionImp>
+class TransferredGlobalFluxFunction;
+
+
+/**
+ * base class for global functions that provides automatic local functions via
+ * LocalizableFunctionInterface
```

Comment:
The most relevant difference to a localizable flux function is that this one can only model flux functions which are _continuous_ w.r.t `x` as well as `u`...